Code/Framework/AzFramework/AzFramework/Asset/AssetSystemComponent.cpp

```diff
@@ -9,24 +9,23 @@
 #include <AzFramework/Asset/AssetSystemComponent.h>
 
 #include <AzCore/Asset/AssetManagerBus.h>
-#include <AzCore/Debug/Profiler.h>
-#include <AzCore/std/string/conversions.h>
 #include <AzCore/Component/ComponentApplicationBus.h>
+#include <AzCore/Debug/Profiler.h>
 #include <AzCore/Interface/Interface.h>
 #include <AzCore/IO/FileIO.h>
 #include <AzCore/IO/IStreamer.h>
 #include <AzCore/IO/Streamer/FileRequest.h>
 #include <AzCore/IO/SystemFile.h>
+#include <AzCore/Settings/SettingsRegistry.h>
 #include <AzCore/std/chrono/chrono.h>
 #include <AzCore/std/string/conversions.h>
-#include <AzCore/StringFunc/StringFunc.h>
 
 #include <AzFramework/API/ApplicationAPI.h>
 #include <AzFramework/Asset/AssetCatalogBus.h>
 #include <AzFramework/Asset/AssetProcessorMessages.h>
 #include <AzFramework/Asset/AssetSeedList.h>
-#include <AzFramework/Asset/NetworkAssetNotification_private.h>
 #include <AzFramework/Asset/Benchmark/BenchmarkCommands.h>
+#include <AzFramework/Asset/NetworkAssetNotification_private.h>
 #include <AzFramework/Network/AssetProcessorConnection.h>
```

Comment:
Authors Note:  Used the visual studio "Sort headers alphabetically" option, the only added one is the Settings registry.